CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to 0-OS
+
+The following sections outline the process all changes to the repositories go through.  
+All changes, regardless of whether they are from newcomers to the community or  
+from the core team follow the same process and are given the same level of review.
+
+- [Team values](#team-values)
+- [Contributing a feature](#contributing-a-feature)
+- [Setting up to contribute](#setting-up-a-development-environment)
+- [Pull requests](#pull-requests)
+- [Issues](#issues)
+
+## Team values
+
+We promote and encourage a set of [shared values](VALUES.md) to improve our
+productivity and inter-personal interactions.
+
+## Contributing a feature
+
+In order to contribute a feature you'll need to go through the following steps:
+
+- Create a Github issue to explain the feature you want to create. A member of the core team will engage with
+you and together you will decide if the feature could be accepted and what is the best approach to take to implement it.
```

Comment:
Should't the label `type_feature` at least set ?

Reply:
normal user cannot set labels on issue. This is only possible for owner/contributors